components/frontend/package-lock.json

```diff
@@ -2480,7 +2480,6 @@
       "resolved": "https://registry.npmjs.org/@tanstack/react-query/-/react-query-5.90.10.tgz",
       "integrity": "sha512-BKLss9Y8PQ9IUjPYQiv3/Zmlx92uxffUOX8ZZNoQlCIZBJPT5M+GOMQj7xislvVQ6l1BstBjcX0XB/aHfFYVNw==",
       "license": "MIT",
-      "peer": true,
```

Comment:
I don't think this is necessary. It is out of scope for the PR and could cause problems with node versions.